### ISSUE TITLE
Make the linux installation experience better

### ISSUE THREAD

**Issue by pierremtb** [KittyCAD/modeling-app#4511]
We're missing any sort of Linux installation instructions, the overall experience isn't polished yet on that platform. On fresh Ubuntu installs I had to install these packages to run the appimage:
```
sudo apt install zlib1g-dev libfuse2 libnss3 libatk1.0-0 libatk-bridge2.0-0 libcups2 libgtk-3-0 libgbm-dev libasound2 xdg-utils
```

**Comment by pierremtb** [comment]
Just found out that there is actually support for .deb autoupdates through https://www.electron.build/electron-updater.class.debupdater, which isn't well documented on their main page (thought it was AppImage only). This could help install the few dependencies that are needed to run and make the overall linux experience better

Think .deb should be the first option to download on linux, with .AppImage the fallback for non debian based folks.

**Comment by TomPridham** [comment]
I was able to run the AppImage once I installed libfuse2, which did require adding the Universe repository. But I didn't need to install any of those other packages on a fresh Ubuntu 22 install. Not sure why you needed them and I didn't.
I updated the `electron-builder.yml` file to also output `.deb` files: https://github.com/KittyCAD/modeling-app/pull/4554. I had a previous commit that added a `deb.depends` array that had those packages in there, but I removed it because it wasn't necessary for me to install the `.deb` file. I'd be happy to document installing `libfuse2` if there is a place to do that. I didn't see anything that felt like a good place in the readme
